### PR TITLE
integrate FAQ answers into docs

### LINKED ISSUE #137: Relocate FAQ content
Remove [FAQ page ](https://docs.goquorum.consensys.net/en/latest/Reference/FAQ/)and make sure every answer is clear from reading the rest of the documentation. Add clarifying content to docs or restructure docs content as needed.

### PR DESCRIPTION
**Pull request by alexandratran**
## Pull Request Description

Remove FAQ and distribute answers throughout docs, ensuring users can find answers in the relevant doc sections.

## Fixed Issue(s)

fixes #137

### CODE REVIEW

**Review by achraf17 — APPROVED** [review]
LGTM

File: docs/Concepts/Privacy/PrivateAndPublic.md [hunk 1]
```diff
@@ -42,21 +42,18 @@ a correction to a bond security definition.
...
-    Otherwise, the transaction doesn't propagate and an error is returned.
+`privateFor` can take multiple addresses in a comma-separated list.
 
 When a GoQuorum node encounters a transaction with a non-null `privateFor` value, it sets the `v` value of the
 transaction signature to `37` or `38` (as opposed to public transactions, whose `v` values are set according to
 [EIP-155](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md)).
 
+!!! note "Notes"
+
+    - `privateFor` is not shared with other participants; it's only used to know which nodes to send the encrypted payload to.
```
Comment (rolandtyler) [inline docs/Concepts/Privacy/PrivateAndPublic.md:53]:
```suggestion
    - `privateFor` is not shared with other participants; it's only used to identify which nodes to send the encrypted payload to.
```

File: docs/HowTo/Configure/Consensus-Protocols/Raft.md [hunk 2]
```diff
@@ -20,15 +19,52 @@ The enode IDs must include a `raftport` querystring parameter specifying the Raf
...
+We recommend having an odd number of at least three nodes in the network.
 
 ## Minting frequency
 
 By default, blocks are minted no more frequently than every 50ms. When transactions arrive:
 
-* If it has been at least 50ms since the last block, a new block is minted immediately.
-* If it has been less that 50ms, a new block is minted 50ms after the previous block was minted. Waiting
+- If it has been at least 50ms since the last block, a new block is minted immediately.
+- If it has been less that 50ms, a new block is minted 50ms after the previous block was minted. Waiting
```
Comment (rolandtyler) [inline docs/HowTo/Configure/Consensus-Protocols/Raft.md:67]:
```suggestion
- If it has been less than 50ms, a new block is minted 50ms after the previous block was minted. Waiting
```

**Review by rolandtyler — APPROVED** [review]
Two suggestions, otherwise LGTM

**Pull request merged**